containers/ddev-webserver/ddev-webserver-base-files/etc/supervisor/conf.d/supervisor.conf

```diff
@@ -1,7 +1,7 @@
 [supervisord]
 logfile=/var/log/supervisord.log ; (main log file;default $CWD/supervisord.log)
 loglevel=info                ; (log level;default info; others: debug,warn,trace)
-pidfile=//var/run/supervisord.pid ; (supervisord pidfile;default supervisord.pid)
+pidfile=/var/run/supervisord.pid ; (supervisord pidfile;default supervisord.pid)
```

Comment:
Same also in the prod folder

Reply:
Of course `//var` or `//anything` or `/anything//something` is all valid in Linux and most linux-like systems.

Reply:
But it also needed the apache fix in prod, so thanks!

Reply:
Yeah I know, it's just superfluous like written in the Slack thread.